openverse_catalog/dags/rawpixel_workflow.py

```diff
@@ -1,49 +1,25 @@
-from datetime import datetime, timedelta
+# airflow DAG (necessary for Airflow to find this file)
+import logging
+from datetime import datetime
 
-from airflow import DAG
-from airflow.operators.python import PythonOperator
 from provider_api_scripts import raw_pixel
-from util.operator_util import get_log_operator
+from util.dag_factory import create_provider_api_workflow
 
 
-DAG_DEFAULT_ARGS = {
-    "owner": "data-eng-admin",
-    "depends_on_past": False,
-    "start_date": datetime(2020, 1, 15),
-    "email_on_retry": False,
-    "retries": 3,
-    "retry_delay": timedelta(days=1),
-}
-
-DAG_ID = "rawpixel_workflow"
-
-
-def get_runner_operator(dag):
-    return PythonOperator(
-        task_id="pull_rawpixel_data",
-        python_callable=raw_pixel.main,
-        depends_on_past=False,
-        dag=dag,
-    )
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s", level=logging.INFO
+)
 
+logger = logging.getLogger(__name__)
 
-def create_dag():
-    dag = DAG(
-        dag_id=DAG_ID,
-        default_args=DAG_DEFAULT_ARGS,
-        start_date=datetime(2020, 1, 15),
-        schedule_interval="@monthly",
-        catchup=False,
-    )
-
-    with dag:
-        start_task = get_log_operator(dag, DAG_ID, "Starting")
-        run_task = get_runner_operator(dag)
-        end_task = get_log_operator(dag, DAG_ID, "Finished")
-
-        start_task >> run_task >> end_task
-
-    return dag
+DAG_ID = "rawpixel_workflow"
 
 
-globals()[DAG_ID] = create_dag()
+globals()[DAG_ID] = create_provider_api_workflow(
+    DAG_ID,
+    raw_pixel.main,
+    start_date=datetime(1970, 1, 1),
```

Comment:
The start_date was previously `datetime(2020, 1, 15)` for this DAG. Maybe the change is irrelevant but is there a reason for it? The same happens with others DAGs here.

Reply:
True, the start_date is mostly irrelevant here:

start_date is partly legacy from the pre-DagRun era. When creating a new DAG, you probably want to set a global start_date for your tasks using default_args. The first DagRun to be created will be based on the min(start_date) for all your tasks.

Any date in the past would work the same.